wordpress-mobile/WordPress-iOS
# Update  to WordPressKit that only uses AlamoFire

This PR updates the version of WordPressKit where we replace AFNetworking with AlamoFire.

How to test:

- Make sure the unit tests are passing.
- Test on the main app using this branch: try/remove_af_networking_from_wpkit
- Because of the amount of places this code is used check several parts of the app:
  - Login on WP.com 
  - Post and page lists;
  - Reader;
  - Site List;
  - Notifications;
  - Editor
  - Media Library, upload multiple images and videos and check memory impact
   - See if there is any slow downs while scrolling around
- Test on a device!
